src/Chip/ChipCompat.cs

```diff
@@ -138,5 +139,351 @@ public virtual Task<ChipReadAttributeResult> GoToTiltValueAsync (ushort tiltValu
 #endif
 
 	}
+
+	[Mac (12,1), Watch (8,3), TV (15,2), iOS (15,2), MacCatalyst (15,2)]
+	public delegate void ChipDeviceConnectionCallback (ChipDevice device, NSError error);
+
+	[Register("CHIPDeviceController", true)]
```

Comment:
No big deal, but maybe we'll want `[Obsolete]` attributes on these classes as well (like we have for other classes in this file)?

Reply:
Good idea - done.